web/docs/integrations/snowflake/README.md

```diff
@@ -10,14 +10,14 @@ Use the [`to_snowflake`](../../tql2/operators/to_snowflake.md) output operator
 at the end of a pipeline to send events to a specific table.
 
 :::info ADBC
-Tenzir Apache Arrow under the hood to represent batches of events. Therefore, we
-can use [Arrow Database Connectivity
-(ADBC)](https://arrow.apache.org/docs/format/ADBC.html) as interface to
-Snowflake, enabling efficient transfer of large datasets. Think of ADBC as the
-columnar equivalent of JDBC/ODBC. Thanks to the existing [Snowflake
-driver](https://arrow.apache.org/adbc/current/driver/snowflake.html), you can
-directly stream events from a Tenzir pipeline to Snowflake using ADBC's *Bulk
-Ingestion* API. For further details on ADBC, read the [introductory blog
+Tenzir uses [Apache Arrow](https://arrow.apache.org) under the hood to encode
+batches of events into a columnar representation. The [Arrow Database
```

Comment:
Do we need to mention this?

Reply:
Otherwise explaining ADBC is pointless. Since it's a differentiator to other systems that just use the Snowflake REST API, it's value-add.

But it's in a callout for a reason, because it's an (important) implementation detail.